build/common.sh

```diff
@@ -38,10 +38,11 @@ source "${KUBE_ROOT}/hack/lib/init.sh"
 
 # Constants
 readonly KUBE_BUILD_IMAGE_REPO=kube-build
-readonly KUBE_BUILD_IMAGE_CROSS_TAG="$(cat "${KUBE_ROOT}/build/build-image/cross/VERSION")"
 
 readonly KUBE_DOCKER_REGISTRY="${KUBE_DOCKER_REGISTRY:-k8s.gcr.io}"
 readonly KUBE_BASE_IMAGE_REGISTRY="${KUBE_BASE_IMAGE_REGISTRY:-k8s.gcr.io/build-image}"
+readonly KUBE_BUILD_IMAGE_CROSS_REGISTRY_AND_NAME="${KUBE_BUILD_IMAGE_CROSS_REGISTRY_AND_NAME:-k8s.gcr.io/build-image/kube-cross}"
```

Comment:
Should `KUBE_BUILD_IMAGE_CROSS_REGISTRY_AND_NAME` be harmonized with `KUBE_CROSS_IMAGE ` in [build/pause/Makefile:43-47](https://github.com/kubernetes/kubernetes/blob/master/build/pause/Makefile#L43-L47)